run/custom_domain_mapping/main.tf

```diff
@@ -17,14 +17,14 @@
 # [START cloudrun_custom_domain_mapping_parent_tag]
 # [START cloudrun_custom_domain_mapping_run_service]
 resource "google_cloud_run_v2_service" "default" {
-  name     = "custom-domain"
+  name     = "your-service-name" # Replace with your desired service
```

Comment:
Please restore the original name, but keep a similar comment. There's no standard, but common examples are more simply "Replace with your service name".